castra/core.py

```diff
@@ -154,6 +163,19 @@ def __setstate__(self, state):
         self.load_meta()
         self.load_partitions()
 
+    def to_dask(self, columns=None):
+        if columns is None:
+            columns = self.columns
+        import dask.dataframe as dd
```

Comment:
Do you really want to import this in the function? It hides that dask is now (an optional) dependency of castra.